ndcube/tests/test_ndcube.py

```diff
@@ -149,10 +149,10 @@ def test_slicing_removed_world_coords(ndcube_3d_ln_lt_l):
 
     sndc = ndc[0, 0, :]
     all_coords = sndc.global_coords._all_coords
-    assert u.allclose(all_coords[lat_key][1], -0.00555556 * u.deg)
-    assert u.allclose(all_coords[lon_key][1], 0.00277778 * u.deg)
-    assert all_coords[lat_key][0] == lat_key
-    assert all_coords[lon_key][0] == lon_key
+    assert isinstance(all_coords["helioprojective"][1], SkyCoord)
```

Comment:
Probably best to set `celestial_key = "helioprojective"` and use that to make it easier to adapt to future changes to this name or if frame of test WCS is changed.